### PR TITLE
Fixed SignalHandlerProcessor

### PR DESCRIPTION
**Pull request by lyrixx**
Before the patch, the signal handler where unregister in the shouldStop method. so basically ... it does not work

### CODE REVIEW

**Comment by odolbeau** [comment]
Nice catch, thanks!
I think you can simply remove those lines, removing the custom signal handler doesn't makes sense IMO.

**Comment by lyrixx** [comment]
@odolbeau Done :ok_hand:

File: src/Swarrot/Processor/SignalHandler/SignalHandlerProcessor.php [hunk 1]
```diff
@@ -67,7 +63,7 @@ public function process(Message $message, array $options)
     {
         $return = $this->processor->process($message, $options);
 
-        if (extension_loaded('pcntl') && $this->shouldStop($options)) {
```
Comment (lyrixx) [inline src/Swarrot/Processor/SignalHandler/SignalHandlerProcessor.php]: I don't understand this code. IMHO this is wrong. We should always return `$return`, isn't ?

Comment (odolbeau) [reply src/Swarrot/Processor/SignalHandler/SignalHandlerProcessor.php]: Right!

**Comment by odolbeau** [comment]
Thanks for those nice improvements!

**Pull request merged**